packages/terra-slide-group/src/terra-dev-site/test/slide-group/AnimatedNonAnimatedSlideGroup.test.scss

```diff
@@ -0,0 +1,12 @@
+:local {
+  .container-style {
```

Comment:
Can we use of the same class that is defined in [SlideGroupDemo.scss](https://github.com/cerner/terra-framework/blob/2fcbf0eb8727d4d15b8c0da41131181555c53c06/packages/terra-slide-group/src/terra-dev-site/doc/example/SlideGroupDemo.scss#L2) here.

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.